### DISCUSSION TITLE
Duplicated labels for the days of the week

### DISCUSSION THREAD

**Discussion by bkyerv** [gpbl/react-day-picker#1643] [Support] [answered]
I am new to this library and therefore tried to read carefully (evidently still struggling to fix it) read the docs before raising this issue. Can someone please help to understand why name of the days are being displayed both in short and long forms like this?

![image](https://user-images.githubusercontent.com/87478959/211813963-74a4968f-5323-468e-9a2e-ea75778fe858.png)

In general what is the best way to set the format and content of the labels? I tried labels property in the component but am getting type errors. While initially I had an impression that this is one of the simplest libraries for date picking functionality but now am getting an impression it is not and has a steeper learning curve that I thought initially.

**Comment by gpbl** [comment, marked as answer]
Hi @bkyerv, this behavior should have happened when the CSS wasn't properly loaded. However, we just released 8.5.0 which will fix this. Thanks for the report!

**Reply by bkyerv** [reply]
Yeah, exactly. I spent a day on trying to fix it thinking it was related to configs until I figured it was related to css)). Thank you for getting back.
